src/codenamet.scp

```diff
@@ -0,0 +1,54 @@
+func CODE_NAME_T_AUTO_ATTACK(from, self, adddam, arg)
+{
+    if (self.ClassName == 'PC'  from.ClassName != 'PC')
+    {
+        return false, 0, 0, 0, 0;
+    }
+
+    if (from.Job != 288)
+    {
+        return false, 0, 0, 0, 0;
+    }
+
+    // Get extra str and dex for attacker and mix them by using exponent
+    // As CNT is a ranged character that use both str and dex. It is more reasonable to add up both value then using the higher one.
+    var str = GET_EXTRA_STR(from);
+    var dex = GET_EXTRA_DEX(from);
+    
+    var mix = Math.floor(((str^2) + (dex^2))^0.5);
```

Comment:
Use `math.pow` instead because the whole codebase does not use the exponential operator, so avoid using it to keep the code consistent
btw GE use [Lua 5.0](https://www.lua.org/manual/5.1/manual.html#8.2) so you can rely on the document to some extent